streaming/src/main/scala/org/apache/spark/streaming/dstream/DStream.scala

```diff
@@ -374,26 +374,53 @@ abstract class DStream[T: ClassTag] (
    * Wrap a body of code such that the call site and operation scope
    * information are passed to the RDDs created in this body properly.
    */
-  protected def createRDDWithLocalProperties[U](time: Time)(body: => U): U = {
+  protected[streaming] def createRDDWithLocalProperties[U](time: Time)(body: => U): U = {
+    createRDDWithLocalProperties(time, makeRDDOpsVisibleInUI = false)(body)
+  }
+
+  /**
+   * Wrap a body of code such that the call site and operation scope
+   * information are passed to the RDDs created in this body properly.
+   */
+  protected[streaming] def createRDDWithLocalProperties[U](
```

Comment:
Instead of adding this method, why not add a new method `makeRDDOpsVisibleInUI` (or a better name)? It's confusing that having two `createRDDWithLocalProperties` methods when implementing a `DStream`: which one should be override? Without looking at the codes in `DStream`, people cannot answer this question.